tests/sqlparser_duckdb.rs

```diff
@@ -68,3 +68,53 @@ fn test_select_wildcard_with_exclude() {
 fn parse_div_infix() {
     duckdb_and_generic().verified_stmt(r#"SELECT 5 // 2"#);
 }
+
+#[test]
+fn test_create_macro() {
+    let _macro = duckdb().verified_stmt("CREATE MACRO schema.add(a, b) AS a + b");
```

Comment:
typically the `_` prefix is used in rust for variables that are not used to suppress the compiler from warning. I think it should be avoided

Maybe you could call this `stmt` or something to avoid clashing with the rust `macro` keyword but also not starting your variable with `_`